client/src-electron/main.ts

```diff
@@ -226,7 +226,24 @@ class ElectronApp {
       ...(debug ? ["--debug"] : []),
     ]
 
-    const childAegis = child_process.spawn("aegis", procArgs, {
+    const venvPath = this.findVenvPython(aegisPath)
+
+    if (!venvPath) {
+      throw new Error("Virtual environment not found")
+    }
+
+    const pythonExec =
+      process.platform === "win32"
+        ? path.join(venvPath, "python.exe")
+        : path.join(venvPath, "python")
+
+    const command = pythonExec
+    const args = [
+      "-c",
+      `import sys; sys.argv = ['aegis'] + ${JSON.stringify(procArgs)}; from aegis_game import main; main()`,
+    ]
```

Comment:
what is this witch craft

Reply:
You can just do this no?

```suggestion
    const aegis =
      process.platform === "win32"
        ? path.join(venvPath, "aegis.exe")
        : path.join(venvPath, "aegis")
```

and then pass in `procArgs` like before normally?

Reply:
Thought this was the first thing I tried and it failed so I went w the hacky thing, I'll look at it again though